models/gs2/README.md

```diff
@@ -0,0 +1,48 @@
+# GS2 Fusion Plasma Simulation
+
+## Overview
+This model uses [GS2](https://gyrokinetics.gitlab.io/gs2/index.html) to simulate plasma in a spherical tokamak. It aims to study various modes in the plasma due to microinstabilities. The current setup terminates once an unstable mode is found. Otherwise, it will continue until reaching a fixed timestep. Therefore, the runtime varies depending on the input parameters.
```

Comment:
Maybe we could make it clear that it is using the gyrokinetic model? There are other plasma simulation frameworks out there, and I think the theoretical perspective is important to plasma physicists.